src/workflow_action.go

```diff
@@ -0,0 +1,45 @@
+package cicero
```

Comment:
Since this is a service it should live in the `service` package, but you probably tried that already (doesn't work because the `Evaluator` is in the `cicero` package). You would have to move the `Evaluator` as well. But we can merge now and do that later.